tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -174,6 +175,12 @@ install_cc_tdx_kernel() {
 	install_cc_tee_kernel "tdx"
 }
 
+install_cc_sev_kernel() {
+	export tee="sev"
+	export kernel_version="$(yq r $versions_yaml assets.kernel.${tee}.version)"
```

Comment:
Why exporting those to the environment ?

Reply:
I guess we don't need to export, indeed.  And I think it came due to a c&p from the TDX work that may also be unnecessarily exporting those (and I will fix that in a different PR).